test/extended/node/cgroups.go

```diff
@@ -0,0 +1,86 @@
+package node
+
+import (
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-node][Serial][Feature:Remove support to configure Cgroup v1 from OCP version >= 4.19]", func() {
+
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLIWithoutNamespace("node").AsAdmin()
+	)
+	const cgroupV2 = "cgroup2fs"
+
+	g.It("Should result in an error when cgroupMode is changed from v2 to v1", func() {
+
+		g.By("1) Check Cluster Version")
+		clusterVersion, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("clusterversion").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		e2e.Logf("Cluster version: %s\n", clusterVersion)
```

Comment:
Looking at the prow job failure, microshift doesn't seem to have `clusterversion` resource installed.
This made me think whether these lines are really necessary to check the cluster version?

Ref: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/29640/pull-ci-openshift-origin-main-e2e-aws-ovn-microshift-serial/1912040806420582400 

```
{  fail [github.com/openshift/origin/test/extended/node/cgroups.go:28]: Unexpected error:
    <*fmt.wrapError | 0xc0065ec960>: 
    Error running /usr/bin/oc --kubeconfig=/tmp/secret/kubeconfig get clusterversion:
    StdOut>
    error: the server doesn't have a resource type "clusterversion"
    StdErr>
    error: the server doesn't have a resource type "clusterversion"
    exit status 1
    
    {
```